packages/@aws-cdk/aws-codebuild/lib/project.ts

```diff
@@ -316,7 +316,7 @@ abstract class ProjectBase extends Resource implements IProject {
    * @param metricName The name of the metric
    * @param props Customization properties
    */
-  public metric(metricName: string, props: cloudwatch.MetricOptions) {
+  public metric(metricName: string, props?: cloudwatch.MetricOptions) {
```

Comment:
```suggestion
  public metric(metricName: string, props: cloudwatch.MetricOptions = { }) {
```

Reply:
If you look at the implementation it's fine.